governance/default.go

```diff
@@ -54,6 +54,11 @@ var (
 		"istanbul.policy":               params.Policy,
 		"istanbul.committeesize":        params.CommitteeSize,
 		"governance.unitprice":          params.UnitPrice,
+		"kip71.lowerboundbasefee":       params.LowerBoundBaseFee,
+		"kip71.gastarget":               params.GasTarget,
+		"kip71.blockgaslimit":           params.BlockGasLimit,
+		"kip71.basefeedenominator":      params.BaseFeeDenominator,
+		"kip71.upperboundbasefee":       params.UpperBoundBaseFee,
```

Comment:
Keys of this map are exposed to node operators, and they should type them to vote. In this context, `kip71` is not intuitive. I think `fee`, `transaction`, `txpool`, or `governance` are more intuitive for those values. How about changing the names?